### PR TITLE
AmeriGEO workshop post

### PR DESCRIPTION
**Pull request by jnywong**
In draft mode pending resolution of #287 and sign-off from AmeriGEO workshop facilitators.

### CODE REVIEW

File: content/blog/2024/amerigeo-workshop/index.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Over 100 participants were able to access the interactive workshop on our infrastructure, with 8 GB of RAM per user to facilitate the processing of large amounts of earth data. We are pleased that the workshop was successful and the platform was able to provide a great experience for participants. After the workshop, Kytt reported that
+
+>The technology worked as expected and we didn't run into 1 major technical problem. Thank you so much for that!".
+
+## Acknowledgements
+
+- Funding from Science Mission Directorate’s Open Source Science Initiative, Research Opportunities in Space and Earth Science (ROSES-2022), F.14 Transform to Open Science
```
Comment (kyttmacmanus) [inline content/blog/2024/amerigeo-workshop/index.md:25]: Should also acknowledge the NASA Socioeconomic Data and Applications Center (SEDAC)

File: content/blog/2024/amerigeo-workshop/index.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+title: Ephemeral Interactive Computing for AmeriGEO Workshop
+date: "2024-08-05"
+banner:
+  image: "featured.png"
```
Comment (kyttmacmanus) [inline content/blog/2024/amerigeo-workshop/index.md:5]: Not a big deal, but if you could grab a screenshot where my face isn't quite so contorted that would be great! If not no worries.

**Review by kyttmacmanus — CHANGES_REQUESTED** [review]
minor comments

File: content/blog/2024/amerigeo-workshop/index.md (hunk 2 above)
Comment (jnywong) [reply content/blog/2024/amerigeo-workshop/index.md:5]: I went back and https://github.com/2i2c-org/2i2c-org.github.io/pull/288/commits/4f495f10f2b9b9f274eb4879fd1465882220eb8f was the best I could do! Video screengrabs of people talking are hard 😅

File: content/blog/2024/amerigeo-workshop/index.md (hunk 1 above)
Comment (jnywong) [reply content/blog/2024/amerigeo-workshop/index.md:25]: Thanks! Resolved in https://github.com/2i2c-org/2i2c-org.github.io/pull/288/commits/ce822360a0861920d41c1a71bfb2069fe9595b83

**Review by kyttmacmanus — APPROVED** [review]
> I went back and https://github.com/2i2c-org/2i2c-org.github.io/pull/288/commits/4f495f10f2b9b9f274eb4879fd1465882220eb8f was the best I could do! Video screengrabs of people talking are hard 😅 

Thanks!

**Review by kyttmacmanus — APPROVED** [review]
Thanks Jenny!

**Pull request merged**